pkgs/top-level/all-packages.nix

```diff
@@ -18488,9 +18488,10 @@ with pkgs;
   electron_22 = electron_22-bin;
   electron_23 = electron_23-bin;
   electron_24 = electron_24-bin;
-  electron_25 = if lib.meta.availableOn stdenv.hostPlatform electron-source.electron_25 then electron-source.electron_25 else electron_25-bin;
+  electron_25 = electron_25-bin;
```

Comment:
Can we please keep this? Lots of things are still using electron 25 like bitwarden and I want to stay as blob free as possible.

Reply:
It would not be built by hydra anyways, since it is EOL and thus marked as insecure

Reply:
Not according to [their documentation](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline).